hack/update-quarkus-platform.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+# This script check whether Quarkus templates use current platform.
+# It's done by querying https://code.quarkus.io/api/platform.
+# If the platform is out of date this scripts create PR with an update.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+LATEST_PLATFORM="$(curl -s "https://code.quarkus.io/api/platforms" | \
+  jq -r '.platforms[0].streams[0].releases[0].quarkusCoreVersion')"
+
+echo "Latest Quarkus platform is: $LATEST_PLATFORM"
+
+XPATH='//*[local-name()="quarkus.platform.version"]/text()'
+
+CE_TEMPLATE_PLATFORM="$(xmllint --xpath "$XPATH" templates/quarkus/cloudevents/pom.xml)"
+HTTP_TEMPLATE_PLATFORM="$(xmllint --xpath "$XPATH" templates/quarkus/cloudevents/pom.xml)"
+
+if [ "$CE_TEMPLATE_PLATFORM" == "$LATEST_PLATFORM" ] && \
+  [ "$HTTP_TEMPLATE_PLATFORM" == "$LATEST_PLATFORM" ]; then
+  echo "Everything is up-to-date!"
+  exit 0
+fi
+
+echo "Quarkus platform in templates is out-of-sync."
+
+PR_BRANCH="update-quarkus-platform-${LATEST_PLATFORM}"
+
+if git fetch origin "$PR_BRANCH" ; then
+  echo "The PR branch already exists!"
+  exit 0
+fi
+
+sed -i -E "s#<quarkus.platform.version>.+</quarkus.platform.version>#<quarkus.platform.version>${LATEST_PLATFORM}</quarkus.platform.version>#g" \
+  ./templates/quarkus/cloudevents/pom.xml
+sed -i -E "s#<quarkus.platform.version>.+</quarkus.platform.version>#<quarkus.platform.version>${LATEST_PLATFORM}</quarkus.platform.version>#g" \
+  ./templates/quarkus/http/pom.xml
+make zz_filesystem_generated.go
+
+git config --global user.email "quarkus-bot@example.com"
+git config --global user.name "quarkus-platform-update-bot"
```

Comment:
I don't think this will get past EasyCLA without getting lawyers involved. Can we make it one of the existing Knative bots instead?

Reply:
I don't know.
Maybe @rhuss or @dsimansk  know more.